File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
function solution(s) {
    let arr = s.split(' ');
    let min = arr[0];
    let max = arr[0];
    arr.forEach(i => {
        if(Number(i) < Number(min)){
            min = i
        }
        if(Number(i) > Number(max)){
            max = i
        }
    })
    return min + " "+ max
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.Arrays;
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] temp = s.split(" ");
        int[] arr = new int[temp.length];
        for(int i = 0; i < temp.length; i++){
            arr[i] = Integer.parseInt(temp[i]);
        }
        Arrays.sort(arr);
        answer += String.valueOf(arr[0]);
        answer += " ";
        answer += String.valueOf(arr[arr.length - 1]);
        return answer;
    }
}